### ISSUE TITLE
Error while install environment

### ISSUE THREAD

**Issue by codomposer** [EthanCloin/issue-tracker-backend#4]
```
Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Preparing editable metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing editable metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [20 lines of output]
      Traceback (most recent call last):
        File "E:\dev\contribute\issue-tracker-backend\env\lib\site-packages\pip\_vendor\pep517\in_process\_in_process.py", line 351, in <module>
          main()
```

**Comment by EthanCloin** [comment]
this project has its dependencies managed by Poetry, try to build by following the steps in the README. Those steps are available in the currently staged PR 👍

**Comment by codomposer** [comment]
Can you add build command step by step in README?

**Comment by EthanCloin** [comment]
Addressed in #3
